templates/CRM/Contribute/Form/Contribution/PremiumBlock.tpl

```diff
@@ -62,7 +62,9 @@
               <div class="premium-full-title">{$row.name}</div>
               <div class="premium-full-disabled">
                 {ts 1=$row.min_contribution|crmMoney}You must contribute at least %1 to get this item{/ts}<br/>
-                <button type="button" value="{ts 1=$row.min_contribution|crmMoney}Contribute %1 Instead{/ts}" amount="{$row.min_contribution}" />
+                <button class="btn btn-secondary" type="button" amount="{$row.min_contribution}">
```

Comment:
I wasn't aware we were putting in bootstrap classes here? should we be using crm-button class instead?

Reply:
Good point. I removed the classes and it still works.